build/redeemer/Redeemer.sol

```diff
@@ -90,6 +92,14 @@ contract Redeemer {
     // Get the principal token that is being redeemed by the user
     address principal = IMarketPlace(marketPlace).markets(u, m, p);
 
+    // Make sure we have the correct principal
+    if (p != uint8(MarketPlace.Principals.Swivel) &&
```

Comment:
so we are gunna go back at some point and use custom errors for all the `require` stuff

but

why is this one in an old-school revert vs a require like `Lender` ?